src/exchange/platform/startExchange.ts

```diff
@@ -23,8 +23,8 @@ const startExchange = (
       await withDevicePromise(deviceId, async (transport) => {
         log("exchange", `attempt to connect to ${deviceId}`);
         if (unsubscribed) return;
-        const swap = new Exchange(transport, exchangeType, 0x00); // TODO Pass fixed/float for UI switch ?
-        const nonce: string = await swap.startNewTransaction();
+        const exchange = new Exchange(transport, exchangeType); // TODO Pass fixed/float for UI switch ?
```

Comment:
You could probably solve that todo if you wanted now, isn't it a matter of adding an optional rateType to the `StartExchagneInput` and passing it in?